src/components/KSlideout/KSlideout.cy.ts

```diff
@@ -20,18 +20,6 @@ describe('KSlideout', () => {
     cy.get('p').should('be.visible')
   })
 
-  it('emits close when panel-background is clicked', () => {
```

Comment:
Why did we remove this test?

Reply:
I removed it because now with the use of `onClickOutside`, even though the Close is emitted, Cypress doesn't gets that emitted value and it fails.